pytroll_collectors/tests/test_harvest_EUM_schedules.py

```diff
@@ -117,7 +117,9 @@ def test_pass_list_file_name(self):
         self.assertEqual(os.path.join(self.basedir, 'ears_viirs_pass_prediction_19-12-16.txt'), save_file)
 
     def test_pass_list_no_sensor(self):
-        params = {'granule_metadata': self.granule_metadata,
+        granule_metadata = self.granule_metadata.copy()
+        granule_metadata.pop('sensor')
```

Comment:
you don't need to copy the metadata, the `setUp` method is rerun before each test.

Reply:
Ah, ok. Let me fix that